src/poggit/module/webhooks/GitHubLoginModule.php

```diff
@@ -58,10 +58,10 @@ public function output() {
 
         $rows = Poggit::queryAndFetch("SELECT opts FROM users WHERE uid = ?", "i", $uid);
         if(count($rows) === 0){
-            Poggit::queryAndFetch("INSERT INTO users (uid, name, token) VALUES (?, ?, ?, '{}')",
-                "isss", $uid, $name, $token);
+            $opts = "{}";
+            Poggit::queryAndFetch("INSERT INTO users (uid, name, token, opts) VALUES (?, ?, ?, ?)",
+                "isss", $uid, $name, $token, $opts);
```

Comment:
I get: RuntimeException: Failed to prepare statement: Column count doesn't match value count